js/apps/admin-ui/src/realm-settings/NewAttributeSettings.tsx

```diff
@@ -151,7 +151,7 @@ export default function NewAttributeSettings() {
         "annotations",
         Object.entries(annotations || {}).map(([key, value]) => ({
           key,
-          value,
+          value: value as Record<string, unknown>,
```

Comment:
Looking at the type, it is the same as the one it is being cast to. So I think we can omit this cast.
```suggestion
          value,
```

Reply:
the build failed when it was omitted

Reply:
Ahhh, I see. I didn't re-build the admin client, so I never got the new type.